src/main.rs

```diff
@@ -1,20 +1,36 @@
-mod data;
 use crossterm::{
     cursor::{self, MoveTo},
     style::Print,
     terminal::{self, Clear, ClearType},
     QueueableCommand,
 };
 use std::io::{stdout, Write};
+use std::iter::zip;
+use std::sync::Arc;
```

Comment:
`cargo fmt`

Reply:
I reordered the uses as ``cargo fmt`` suggests, but I don't agree with the others, it breaks my logic too much. In case you insist on using ``cargo fmt`` for the entire project, however, that's fine by me, even if I personally don't like the changes it proposes.